composer.json

```diff
@@ -3,6 +3,10 @@
     "description": "This tool check syntax of PHP files about 20x faster than serial check.",
     "homepage": "https://github.com/php-parallel-lint/PHP-Parallel-Lint",
     "license": "BSD-2-Clause",
+    "keywords": [
+        "lint",
+        "static analysis"
```

Comment:
`static analysis` is a magic keyword, which makes composer double check, when a project adds `php-parallel-lint/php-parallel-lint` as a dependency that it might wanne do so as a `require-dev`-dependency